.github/workflows/build.yml

```diff
@@ -45,8 +45,6 @@ jobs:
       - name: Test with Maven
         run: mvn test --batch-mode
       - name: Run Sonar analysis
-        # TODO: change only when Sonar Analyze is disabled for jhipster/jhipster
-        if: github.repository == 'jhipster/jhipster'
```

Comment:
you need to keep this, and change the repo to jhipster/jhipster-bom
Otherwise, it will try to trigger the analyze to fork projects

Reply:
@pascalgrimaud It is already declared on top:
![image](https://user-images.githubusercontent.com/3706415/97807945-99a10380-1c5b-11eb-978c-19bd512ca2cd.png)

Reply:
Oh ok, good :eye: :eye: !